biz/auth.manager.js

```diff
@@ -19,7 +19,73 @@ class AuthManager extends BaseManager {
     this._authRepository = new AuthRepository();
   }
 
-  async signUp(bodyParams) {
+  async merchantSignup(bodyParams) {
```

Comment:
add a another param in function and fetch schema. pass the schema from controller so that you can use same login and singup function. Do the required changes in auth repository as well!